libs/wingsdk/src/shared-aws/api.onrequest.inflight.ts

```diff
@@ -1,3 +1,4 @@
+import querystring from "querystring";
```

Comment:
Should we use `URLSearchParams` instead?

Reply:
makes sense since it seems to be the new way of doing it 😃